installer/tue-install-impl.bash

```diff
@@ -375,47 +375,74 @@ function tue-install-add-text
         tue-install-error "Invalid tue-install-add-text call. Usage: tue-install-add-text SOURCE_FILE TARGET_FILE"
     fi
 
-    local source_file=$TUE_INSTALL_CURRENT_TARGET_DIR/$1
+    tue-install-debug "tue-install-add-text $*"
+
+    local source_file=$1
+    if [[ "$source_file" != "/"* ]] || [[ "$source_file" != "~/"* ]]
+    then
+        tue-install-debug "tue-install-add-text: relative source file"
+        source_file="$TUE_INSTALL_CURRENT_TARGET_DIR/$source_file"
+    else
+        tue-install-debug "tue-install-add-text: absolute source file"
+    fi
     local target_file=$2
+    if [[ "$target_file" != "/"* ]] && [[ "$source_file" != "~/"* ]]
+    then
+        tue-install-error "tue-install-add-text: target file needs to be absolute or relative to the home directory"
+    fi
 
     local root_required=true
     if namei -l $target_file | grep -q $(whoami)
     then
+        tue-install-debug "tue-install-add-text: NO root required"
```

Comment:
Check whether root is required to read source file as the source file is not restricted to just the targets repo